client/src/components/Ticketing/GroupToggle.tsx

```diff
@@ -11,6 +11,9 @@ const ShowingActivenessToggle =(props) => {
     newFilterSetting: string,
   ) => {
     setFilter(newFilterSetting);
+    if (props.handleFilterChange) {
+      props.handleFilterChange(newFilterSetting);
+    }
   };
 
   return (
```

Comment:
I noticed that each button in the group is missing a right border. I think this might be able to be resolved by moving all of the `backgroundColor: white` to the `sx` prop of the `ToggleButtonGroup`

Reply:
fixed